project-management-openproject/project-management-openproject-api/src/main/java/com/xwiki/projectmanagement/openproject/internal/OpenProjectClient.java

```diff
@@ -109,7 +114,10 @@ public PaginatedResult<WorkItem> getWorkItems(int page, int pageSize, List<LiveD
                 workPackagesPaginatedResult,
                 OpenProjectConverters::convertWorkPackageToWorkItem
             );
-        } catch (Exception e) {
+        } catch (WorkItemRetrievalBadRequestException e) {
+            logger.error(e.getMessage());
```

Comment:
i think a warning where you specify what happened and the root cause message wold be more appropriate